packages/opa/README.md

```diff
@@ -1,331 +1,296 @@
-# @styra/opa
+# OPA Typescript SDK
 
-<div align="left">
-    <a href="https://speakeasyapi.dev/"><img src="https://custom-icon-badges.demolab.com/badge/-Built%20By%20Speakeasy-212015?style=for-the-badge&logoColor=FBE331&logo=speakeasy&labelColor=545454" /></a>
-    <a href="https://opensource.org/licenses/MIT">
-        <img src="https://img.shields.io/badge/License-MIT-blue.svg" style="width: 100px; height: 28px;" />
-    </a>
-</div>
+The Styra-supported driver to connect to Open Policy Agent (OPA) and Enterprise OPA deployments.
 
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![NPM Version](https://img.shields.io/npm/v/%40styra%2Fopa?style=flat&color=%2324b6e0)](https://www.npmjs.com/package/@styra/opa)
+[![JSR](https://jsr.io/badges/@styra/opa)](https://jsr.io/@styra/opa)
 
-## 🏗 **Welcome to your new SDK!** 🏗
+> The documentation for this SDK lives at https://docs.styra.com/sdk, with reference documentation available at https://styrainc.github.io/opa-typescript
 
-It has been generated successfully based on your OpenAPI spec. However, it is not yet ready for production use. Here are some next steps:
-- [ ] 🛠 Make your SDK feel handcrafted by [customizing it](https://www.speakeasyapi.dev/docs/customize-sdks)
-- [ ] ♻️ Refine your SDK quickly by iterating locally with the [Speakeasy CLI](https://github.com/speakeasy-api/speakeasy)
-- [ ] 🎁 Publish your SDK to package managers by [configuring automatic publishing](https://www.speakeasyapi.dev/docs/advanced-setup/publish-sdks)
-- [ ] ✨ When ready to productionize, delete this section from the README
+You can use the Styra OPA SDK to connect to [Open Policy Agent](https://www.openpolicyagent.org/) and [Enterprise OPA](https://www.styra.com/enterprise-opa/) deployments.
 
-<!-- Start SDK Installation [installation] -->
 ## SDK Installation
 
 ### NPM
 
 ```bash
-npm add https://gitpkg.now.sh/StyraInc/ts-packages/packages/opa
+npm add @styra/opa
 ```
 
 ### PNPM
 
 ```bash
-pnpm add https://gitpkg.now.sh/StyraInc/ts-packages/packages/opa
+pnpm add @styra/opa
 ```
 
 ### Bun
 
 ```bash
-bun add https://gitpkg.now.sh/StyraInc/ts-packages/packages/opa
+bun add @styra/opa
 ```
 
 ### Yarn
 
 ```bash
-yarn add https://gitpkg.now.sh/StyraInc/ts-packages/packages/opa zod
+yarn add @styra/opa zod
 
 # Note that Yarn does not install peer dependencies automatically. You will need
 # to install zod as shown above.
 ```
-<!-- End SDK Installation [installation] -->
+<!-- No SDK Installation [installation] -->
```

Comment:
ℹ️ We had to opt-out for now: since SE only generates install instructions using npmjs.com when it itself is in charge of publishing, we'll have to resort to this.